.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
 
 os:
   - linux
+  - osx
```

Comment:
Only stable compiler on OS X needs to be tested.

Reply:
Done.